src/main/java/uk/gov/hmcts/reform/pip/model/publication/ListType.java

```diff
@@ -48,7 +48,8 @@ public enum ListType {
                                         SSCS_DAILY_LIST, "SSCS Daily List - Additional Hearings", false, true),
     IAC_DAILY_LIST(VENUE, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES),
     CARE_STANDARDS_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES),
-    PRIMARY_HEALTH_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES);
+    PRIMARY_HEALTH_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES),
+    OPA_PRESS_LIST(VENUE, CRIME_IDAM, ALL_VERIFIED_THIRD_PARTY_CRIME_ROLES, null, null, true, false);
```

Comment:
I haven't checked if it would cause an issue, but wondering if we should set 'Excel' to false for now, and enable as part of the other ticket

Reply:
Removed the hasExcel flag for now until the third ticket